journal-of-reconstructive-microsurgery.csl

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Journal of Reconstructive Microsurgery</title>
+    <title-short>J Reconstr Microsurg</title-short>
+    <id>http://www.zotero.org/styles/journal-of-reconstructive-microsurgery</id>
+    <link href="http://www.zotero.org/styles/journal-of-reconstructive-microsurgery" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-medical-association" rel="template"/>
+    <link href="https://www.thieme.de/de/journal-reconstructive-microsurgery/authors-9770.htm" rel="documentation"/>
+    <author>
+      <name>Roland Bockmann</name>
+      <email>go2roland@gmx.net</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <summary>A Variation of the American Medical Association style as used in J Reconstr Microsurg.</summary>
+    <updated>2015-06-04T20:56:26+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
```

Comment:
delete these lines 21 - 25. They set the page-range delimiter to a hyphen, the journal uses en dash (as it should!).